src/compute_horde_prompt_gen/prompt.py

```diff
@@ -4,17 +4,17 @@
 
 class PromptGeneratingPrompt:
     def random_select(self, arr: list[str], num: int = 5) -> str:
-        random.shuffle(arr)
-        return ", ".join(arr[:num]) + ", etc"
+        return random.choices(arr, k=num)
```

Comment:
```suggestion
        return random.sample(arr, k=num)
```
I think we should not pick the same thing twice.